flaml/nlp/result_analysis/azure_utils.py

```diff
@@ -340,43 +333,53 @@ def _get_complete_connection_string(self):
                + self._azure_key + ";EndpointSuffix=core.windows.net"
 
     def _init_azure_clients(self):
+        from azure.storage.blob import ContainerClient
         connection_string = self._get_complete_connection_string()
-        container_client = ContainerClient.from_connection_string(conn_str=connection_string,
-                                                                  container_name=self._container_name)
-        return container_client
+        try:
+            container_client = ContainerClient.from_connection_string(conn_str=connection_string,
+                                                                      container_name=self._container_name)
+            return container_client
+        except ValueError:
+            print("Container name not specified")
+            return None
 
     def _init_blob_client(self,
                           local_file_path):
+        from azure.storage.blob import BlobServiceClient
+
         connection_string = self._get_complete_connection_string()
         blob_service_client = BlobServiceClient.from_connection_string(connection_string)
-        blob_client = blob_service_client.get_blob_client(container=self._container_name, blob=local_file_path)
-        return blob_client
+        try:
+            blob_client = blob_service_client.get_blob_client(container=self._container_name, blob=local_file_path)
+            return blob_client
+        except ValueError:
+            print("Container name or local_file_path are incorrect")
```

Comment:
more specific

Reply:
changed to "_container_name is unspecified or wrongly specified, please specify _container_name in AzureUtils"

Reply:
How about printing self._container_name?